examples/basic_voice.py

```diff
@@ -121,10 +121,13 @@ async def ensure_voice(self, ctx):
         elif ctx.voice_client.is_playing():
             ctx.voice_client.stop()
 
```

Comment:
There should be a note saying that message content is required for prefixed commands, to be consistent with the rest of the examples.